tests/utils/matching/test_matching.py

```diff
@@ -185,6 +185,6 @@ def test_assert_output_matches_multipart_body_with_separator():
 
 def test_assert_output_matches_multiple_separators():
     assert_output_matches(
-        MESSAGE_SEPARATOR + MESSAGE_SEPARATOR + 'AAA' + MESSAGE_SEPARATOR + MESSAGE_SEPARATOR,
+        f'{MESSAGE_SEPARATOR}{MESSAGE_SEPARATOR}AAA{MESSAGE_SEPARATOR}{MESSAGE_SEPARATOR}',
```

Comment:
Here I prefer the original version. I find it much easier to scan :)

Reply:
That's right :+1: